### ISSUE TITLE
Some stock Tickers(NSE & BSE) are missing in Timeseries AlphaVantage module

### ISSUE THREAD

**Issue by Pranav4399** [DidierRLopes/GamestonkTerminal#61]
Hello ! Kudos to great work. Currently I'm going through all the code to understand the terminal. I live in india and We have two stock markets - NSE & BSE. I couldn't find any list of stock tickers from india which are available in this module ? Can you share the list of available shares which are available in Timeseries AlphaVantage module if they're availalbe with you ? Else Can you suggest any other free API which i can use instead of alphavantage ?

**Comment by DidierRLopes** [comment]
Hey @Pranav4399,

Thank you for feedback.

Yes, you're right. I just checked and AlphaVantage doesn't seem to be the best for share prices outside NYSE/NASDAQ, as I saw some people asking for such "list".

I'll try to implement a fallback mechanism to get historical data from other sources in case they aren't available from AV. It may be worth looking into Yahoo Finance, or IEX.

**Comment by aia** [comment]
We should add alternative APIs for market data acquisition

**Comment by DidierRLopes** [comment]
I've implemented this with Yahoo Finance in #128 (the data is acquired from quandl)

Can you let me know if the tickers you usually use are available? 

I tested some of them, some work some don't. The ones that do, still aren't available on Alpha Vantage. So regardless it's an improvement. The others that I've tried and don't work, I get the message from yahoo finance API:
```
load 20MICRONS

1 Failed download:
- 20MICRONS: No data found, symbol may be delisted
```

**Comment by DidierRLopes** [comment]
I'll close this issue for now. If Yahoo Finance is still not good enough, feel free to open a new issue!
